src_common/common_utils.h

```diff
@@ -35,6 +35,10 @@ static const char HEXDIGITS[] = "0123456789abcdef";
 
 #define ARRAY_SIZE(a) (sizeof(a) / sizeof(a[0]))
 
+extern size_t strlcpy(char *dst, const char *src, size_t size);
+
+extern size_t strlcat(char *dst, const char *src, size_t size);
```

Comment:
Why is this needed ?
I think this will cause an issue for plugins, the `src_common` dir is shared with them

Reply:
Without such, I still have a compilation warning, because the function declaration is unknown... :thinking: 
Do you see another way to declare them? There are unknown in the sdk either...

Reply:
this is part of `string.h` with our toolchain version I think, so should be replace with a #include ?

Reply:
Seems no more needed... I removed these lines.